core/tauri/src/endpoints/window.rs

```diff
@@ -124,13 +124,23 @@ impl Cmd {
           let url = options.url.clone();
           let pending =
             crate::runtime::window::PendingWindow::with_config(options, label.clone(), url);
-          window.create_window(pending)?.emit_others_internal(
-            "tauri://window-created".to_string(),
+
+          let raw_event = "tauri://window-created";
+          let event: M::Event = raw_event.parse().unwrap_or_else(|_| {
```

Comment:
should this use `tauri_event` fn?

Reply:
ah yeah, i made that before i did that helper fn